v2/internal/pkg/operator/filtered_collector.go

```diff
@@ -547,3 +553,20 @@ func (o FilterCollector) isAlreadyFiltered(ctx context.Context, srcImage, filter
 	}
 	return filteredImageDigest == catalogDigest
 }
+
+// isDeleteOfV1CatalogFromDisk returns true when trying to delete an operator catalog mirrored by oc-mirro v1 and the catalog was on disk (using oci:// on the ImageSetConfiguration)
+// TODO remove me when the migration from oc-mirror v1 to v2 ends
+func (o *FilterCollector) isDeleteOfV1CatalogFromDisk(isOperatorCatalogFromDisk bool) bool {
+	return o.Opts.IsDiskToMirror() && o.Opts.IsDelete() && o.generateV1DestTags && isOperatorCatalogFromDisk
```

Comment:
Nit:
```suggestion
func (o *FilterCollector) isDeleteOfV1CatalogFromDisk() bool {
	return o.Opts.IsDiskToMirror() && o.Opts.IsDelete() && o.generateV1DestTags
```
Very minor and no need to change it as we might get rid of this altogether in the future, but there is no point in passing in a boolean that you already know renders the whole call `false`.

Reply:
Good one, thanks, I will change it here.